azerothcore/azerothcore-wotlk
# fix(Core/Spell): rogue's poisons logic check

Co-Authored-By: Yehonal yehonal.azeroth@gmail.com

## CHANGES PROPOSED:

-  Fix logic in an `if` block where a condition was always returning true

![image](https://user-images.githubusercontent.com/75517/89887565-d74b3e80-dbce-11ea-9efb-46b3d13ed7a0.png)


## TESTS PERFORMED:

- Build, runs



## HOW TO TEST THE CHANGES:

- Use a Rogue character and play with Poisons. Make sure they are still correctly applied in all cases (hit, crit, etc..)
- Play a little bit with spells and hitting in general, make sure nothing is broken by this change.

## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
